src/citrine/_rest/resource.py

```diff
@@ -21,6 +22,7 @@ class ResourceTypeEnum(BaseEnumeration):
 
     TEAM = "TEAM"
     PROJECT = "PROJECT"
+    BRANCH = "BRANCH"
```

Comment:
I think BRANCH is actually just a MODULE under the hood (like the rest of ai engine assets)

Reply:
Depending what you mean by module, yes. We don't differentiate them in the accounts service. That may be a mistake though, not sure what it's used for.

Reply:
Sure.

For my own info, what is ResourceTypeEnum used for?